static/sass/takeovers/_1710-release.scss

```diff
@@ -0,0 +1,34 @@
+@mixin p-takeover-1710-release {
+  .p-takeover--1710-release {
+    width: auto;
```

Comment:
Sort CCS rules alphabetically. (The linter will catch this in future so we should keep it in mind)